package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@purple/phoenix-components",
-	"version": "4.4.0",
+	"version": "4.4.1",
```

Comment:
Did you update the version manually or was it automated? Since we use standard-version, it automatically (when executed) changes version of package based on commits since previous release. So if you did that manually, you can revert this change.